File: loopsFoundation/sumOfOdd.cpp
#include<iostream>
using namespace std;
int main()
{
    int number, sum;
    cout<<"Enter a number : ";
    cin>>number;

    for(int i = 1; i <= number; i++)
    {
        if(i % 2 != 0)
        {
            cout<<i<<" ";
        }
        sum = i*i;
    }
    // sum = number * number;
    cout<<endl<<"Sum of "<<number<<" odd numbers = "<<sum;
}

File: loops foundation/sum of odd.py
takeNum = int(input("enter the number :"))
i = 1
sum = 0

while i <= takeNum:
    if i % 3 == 0:
        print(i)
        sum = sum + i
    i = i + 1


print(f"THE SUM OF NUMBERS DIVISIBLE BY 3 IS {sum}")